runtime/hydradx/src/evm/erc20_currency.rs

```diff
@@ -271,34 +272,40 @@ where
 		amount: Self::Balance,
 	) -> sp_runtime::DispatchResult {
 		let sender = <pallet_evm_accounts::Pallet<T>>::evm_address(from);
-
-		// let's construct standard transfer
-		let erc20_transfer = || -> DispatchResult {
-			<Self as ERC20>::transfer(
-				CallContext {
-					contract,
-					sender,
-					origin: sender,
-				},
-				EvmAccounts::<T>::evm_address(to),
-				amount,
-			)
-		};
-
-		// And handle the transfer according to the token type
-		if AaveTradeExecutor::<T>::is_atoken(contract) {
-			AaveTradeExecutor::<T>::transfer(contract, from, to, amount, erc20_transfer)
-		} else {
-			erc20_transfer()
-		}
+		<Self as ERC20>::transfer(
```

Comment:
there was a different code path if the asset is atoken ? 

why removed ?

Reply:
this was actually cause of issue we hotfixed, this is just removing unused code